### ISSUE TITLE
[LV2] some preset parameters are not saved when the plugin state is saved

### ISSUE THREAD

**Issue by alex-tee** [mtytel/helm#260]
1. choose CM Kleer Arp
2. (in ardour) copy the plugin and paste it somewhere else
3. the filter graph looks different in the pasted plugin

same happens in Zrythm, so I think the filter value changes are not communicated with the host

**Comment by xard-dev** [comment]
Yes, I tested this with Carla and can confirm that when loading session with CM Kleer Arp that the filter graphic is not displayed correctly when loading the project. 

I noticed that the upper right corner still says **SH** all times and otherwise the filter controls behave and sound as expected:
![image](https://user-images.githubusercontent.com/65303522/96374397-31dabb00-117b-11eb-9863-ee09c4e913a8.png)

**Comment by xard-dev** [comment]
Okay, I fixed it as I noticed that the filter style is not set during initialization of the UI: Added a piece of code to the resize function which also sets the style for the filter and now it seems to work.
